src/replica/IngestHttpSvcMod.cc

```diff
@@ -80,7 +80,7 @@ json IngestHttpSvcMod::_syncProcessRequest() const {
 
 json IngestHttpSvcMod::_asyncSubmitRequest() const {
     debug(__func__);
-    checkApiVersion(__func__, 13);
+    checkApiVersion(__func__, 15);
```

Comment:
Good point. I already thought about adding some internal bookkeeping (and code documentation) mechanisms for the version numbers within the code. It would be easier to track (and document) REST services where the API version changes.

I will do this in a separate ticket. In the meantime, I have registered the idea in my TODO list for code improvement/refactoring https://confluence.lsstcorp.org/display/DM/Improved+version+number+handling+in+the+REST+API